### ISSUE TITLE
[Recovery] Get status of email registration

### ISSUE THREAD

**Issue by usame-algan** [safe-global/safe-wallet-monorepo#3291]
## Requirements

- [x] Show a new "Email notification" section on the settings page ([Figma](https://www.figma.com/file/pO43pjVwR34Uy1ihMsY3OM/Recovery-Hub?type=design&node-id=1799-33645&mode=design&t=NtH0QPz1xNS6It06-0))
- [x] Clicking the "Sign to view" button should fetch the GET endpoint of the CGW
- [x] If there is an email registered it should be displayed ([Figma](https://www.figma.com/file/pO43pjVwR34Uy1ihMsY3OM/Recovery-Hub?type=design&node-id=1799-35143&mode=design&t=NtH0QPz1xNS6It06-0))
- [x] If there is no email registered there should be a "No email found" message

**Comment by usame-algan** [comment]
This can be tested here https://github.com/safe-global/safe-wallet-web/pull/3331

**Comment by liliya-soroka** [comment]
@usame-algan , should " Clicking the "Sign to view" button should fetch the GET endpoint of the CGW " be available for non-owner? 
In the latest version , I can sign as non-owner and see the email status

**Comment by usame-algan** [comment]
Good point! This will be added with https://github.com/safe-global/safe-wallet-web/issues/3320 so only owners can press that button.

**Comment by liliya-soroka** [comment]
Verified.
Note: state for the owner with and without signing will be checked together with https://github.com/safe-global/safe-wallet-web/issues/3320

**Comment by usame-algan** [comment]
Closing this as the project was removed from the roadmap
